### PR TITLE
Discussion about the translations

### PR DESCRIPTION
**Pull request by Berrysoft**
Although most of the difference is not that necessary, some of them are vital:

* Disassemble: it means "decompile to IL", I think it should be specified in translation, because "反汇编" usually means "decompile to ASM".
* FQ name: it means "fully qualified name", aka "完全限定名称"
* As string: it means the value is treated or showed as string
* Base 10: "十进制", of course
* "Add" method: method name doesn't need to be translated. It also applies to "unmanaged", which is a keyword.
* New tab: it is a dummy name of a new tab, not an action
* Base types: it is a list of the base types of the current type, not basic types.

### CODE REVIEW

File: ILSpy/Properties/Resources.zh-Hans.resx [hunk 1]
```diff
@@ -499,16 +499,16 @@
     <value>在此步骤之后显示状态</value>
   </data>
   <data name="DebugThisStep" xml:space="preserve">
-    <value>调试此步骤</value>
+    <value>调试此步</value>
```
Comment (maikebing) [inline ILSpy/Properties/Resources.zh-Hans.resx]: 调试次步 ， 总感觉读起来有点拗口。

File: ILSpy/Properties/Resources.zh-Hans.resx [hunk 2]
```diff
@@ -475,19 +475,19 @@
     <value>字体:</value>
   </data>
   <data name="Size" xml:space="preserve">
-    <value>大小:</value>
+    <value>大小：</value>
   </data>
   <data name="DebugSteps" xml:space="preserve">
-    <value>调试步骤</value>
+    <value>调试步</value>
```
Comment (maikebing) [inline ILSpy/Properties/Resources.zh-Hans.resx]: 斟酌一下？

File: ILSpy/Properties/Resources.zh-Hans.resx (hunk 1 above)
Comment (Berrysoft) [reply ILSpy/Properties/Resources.zh-Hans.resx]: 嗯我也比较纠结，但是 step 好像也不是“步骤”

File: ILSpy/Properties/Resources.zh-Hans.resx [hunk 3]
```diff
@@ -861,28 +861,28 @@
...
   <data name="DEBUGDumpPDBAsXML" xml:space="preserve">
-    <value>DEBUG -- 转存PDB为XML</value>
+    <value>调试 -- PDB 转储为 XML</value>
   </data>
   <data name="DecompilerSettings.NativeIntegers" xml:space="preserve">
     <value>使用 nint/nuint 类型</value>
   </data>
   <data name="BaseTypes" xml:space="preserve">
-    <value>基础类型</value>
+    <value>基类型</value>
```
Comment (maikebing) [inline ILSpy/Properties/Resources.zh-Hans.resx:886]: 基类好一点还是基类型 ？

File: ILSpy/Properties/Resources.zh-Hans.resx [hunk 4]
```diff
@@ -154,19 +154,19 @@
...
   <data name="OpenFrom_GAC" xml:space="preserve">
-    <value>从_GAC打开...</value>
+    <value>从 GAC 中打开(_G)...</value>
   </data>
   <data name="ReloadAssemblies" xml:space="preserve">
     <value>重新加载全部程序集</value>
   </data>
   <data name="DEBUGDecompile" xml:space="preserve">
-    <value>DEBUG  - 全部反编译</value>
+    <value>调试 -- 全部反编译到 C#</value>
```
Comment (maikebing) [inline ILSpy/Properties/Resources.zh-Hans.resx:163]: 因为能反编译的不止是C#， 还可以反编译到别的语言， 我觉得指定C#， 可能不妥。

File: ILSpy/Properties/Resources.zh-Hans.resx (hunk 3 above)
Comment (Berrysoft) [reply ILSpy/Properties/Resources.zh-Hans.resx:886]: 这个选项的下面就是“派生类型”，另外 BaseTypes 还包括了接口，说是“类”不合适

File: ILSpy/Properties/Resources.zh-Hans.resx (hunk 4 above)
Comment (Berrysoft) [reply ILSpy/Properties/Resources.zh-Hans.resx:163]: 在代码里面就是反编译到C#

File: ILSpy/Properties/Resources.zh-Hans.resx (hunk 1 above)
Comment (maikebing) [reply ILSpy/Properties/Resources.zh-Hans.resx]: VS之类的翻译， 有个“ 单步调试”   之类的，  他这个“步” 有点指代  此行代码的意思。

Comment (Berrysoft) [reply ILSpy/Properties/Resources.zh-Hans.resx]: 我放弃了，步骤就步骤吧，实在没有别的代替了

**Comment by Berrysoft** [comment]
Update: content of about page.

File: ILSpy/Properties/Resources.zh-Hans.resx (hunk 4 above)
Comment (maikebing) [reply ILSpy/Properties/Resources.zh-Hans.resx:163]: image
这里可以选择， 所以！

Comment (Berrysoft) [reply ILSpy/Properties/Resources.zh-Hans.resx:163]: https://github.com/maikebing/ILSpy/blob/4bc5b2bcbb62fd42655312ac5b82522a45b79cce/ILSpy/Commands/DecompileAllCommand.cs#L53

这里用的是 `CSharpLanuage` ，选不了的。如果要 IL with C#，需要 `CSharpILMixedLanguage`

**Pull request merged**